taxonomies/ingredients.txt

```diff
@@ -20318,7 +20319,7 @@ sv:smakberedning
 # usage:fr: préparation aromatique (arôme naturel, colorants : caramel, curcumine, extrait de paprika, céleri)
 
 <en:flavouring
-en:natural flavouring, natural flavour, natural flavor, natural flavoring, natural flavourings, natural flavorings, natural flavours, natural flavors, natural aroma, natural aromas
+en:natural flavouring, natural flavour, natural flavor, natural flavoring, natural flavourings, natural flavorings, natural flavours, natural flavors, natural aroma, natural aromas, natural and nature identical flavouring
```

Comment:
nature identical flavouring is not the same as natural flavouring, see https://aurigaresearch.com/natural-artificial-flavours-fssai-regulations-say/